src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -7,5 +7,20 @@ export default function PrivateRoute({ children, ...rest }) {
   // TODO: Use the user in context to determine whether to redirect to /login
   // const { user } = useUser();
 
-  return <Route {...rest} render={({ location }) => children} />;
+  const { user } = useUser();
+
```

Comment:
Great job setting the user to a variable the determining below where to render the user based on if there is a user or not